Components/Alert/alert.html

```diff
@@ -0,0 +1,77 @@
+<!DOCTYPE html>
+<html lang="en">
+    <head>
+        <meta charset="UTF-8" />
+        <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+        <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+        <title>CandelCSS Alert Component</title>
+        <link rel="stylesheet" href="/assets/css/components.css" />
+        <script
+            src="https://kit.fontawesome.com/dc21c76de2.js"
+            crossorigin="anonymous"
+        ></script>
+    </head>
+    <body>
+        <!-- ALERT -->
+
+        <section>
+            <h1>ALERT</h1>
+
+            <!-- SIMPLE ALERT -->
+
+            <div class="container">
+                <div class="alert primary">A simple primary alert!</div>
+
+                <div class="alert secondary">A simple secondary alert!</div>
+
+                <div class="alert success">A simple success alert!</div>
+
+                <div class="alert danger">A simple danger alert!</div>
+
+                <div class="alert warning">A simple warning alert!</div>
+
+                <div class="alert info">A simple info alert!</div>
+
+                <div class="alert light">A simple light alert!</div>
+
+                <div class="alert dark">A simple dark alert!</div>
```

Comment:
where is the CSS code for all the classes for example primary, secondary, success, etc?

Reply:
The CSS for common classes are there in the components.css file, Zoef.